### PR TITLE
Sync Lock: Don't seek phase when disabling sync

### PR DESCRIPTION
**Pull request by ywwg**

### CODE REVIEW

**Comment by Holzhaus** [comment]
Ist the main target branch correct? Or does this affect 2.3?

File: src/engine/enginebuffer.h [hunk 1]
```diff
@@ -389,6 +393,7 @@ class EngineBuffer : public EngineObject {
     QAtomicInt m_iEnableSyncQueued;
     QAtomicInt m_iSyncModeQueued;
     ControlValueAtomic<QueuedSeek> m_queuedSeek;
+    bool m_previousBufferSeek = false;
```
Comment (daschuer) [inline src/engine/enginebuffer.h:396]: If this is accessed from multiple threads it should be an atomic for proper memory barriers.

**Review by daschuer — CHANGES_REQUESTED** [review]
The fix itself works and locks good. Thank you. 
The issue in 2.3 is slightly different.

File: src/engine/enginebuffer.h (hunk 1 above)
Comment (ywwg) [reply src/engine/enginebuffer.h:396]: It isn't, this value is only read/written in the processSeek function

**Comment by ywwg** [comment]
anything more needed here?

**Review by daschuer — APPROVED** [review]
No, Thank you. LGTM

**Pull request merged**